libs/types-common/src/Data/Misc.hs

```diff
@@ -322,3 +326,15 @@ instance FromJSON PlainTextPassword where
 instance Arbitrary PlainTextPassword where
   -- TODO: why 6..1024? For tests we might want invalid passwords as well, e.g. 3 chars
   arbitrary = PlainTextPassword . fromRange <$> genRangeText @6 @1024 arbitrary
+
+--------------------------------------------------------------------------------
+-- ModJulianDay
+
+newtype ModJulianDay = ModJulianDay {fromUTCDay :: Day}
```

Comment:
Mod for "Modified Julian Day". Didn't want to nameclash with [Data.Time.Calendar](https://hoogle.zinfra.io/file/root/.ghcup/ghc/8.8.4/share/doc/ghc-8.8.4/html/libraries/time-1.9.3/Data-Time-Calendar.html#t:Day). Also this newtype was just me sidestepping the issue of writing a general`Cql` instances for `Day`. Now since we know that `CqlDate Int32` exist it might be worth writing these instances.